ios/RNFirebase/fabric/crashlytics/RNFirebaseCrashlytics.m

```diff
@@ -46,7 +46,7 @@
 
             if(dict[@"additional"]){
                 CLSStackFrame *f = [CLSStackFrame stackFrame];
-                [f setLibrary: "Additional Parameters"];
+                [f setLibrary: @"Additional Parameters"];
```

Comment:
I was literally just cooking up a PR for this! It slipped in because crashlytics wasn't available in CI, which is exactly what you fixed above. Double thank you

Reply:
wwwwwuuuuttt
you're so fast

Reply:
thanks for attention ;)

Reply:
Well, I was the one that did the release with the error, and now I'm integrating it into my main work project, so more just that I should be sleeping but I'm not ;-).

Reply:
I know that feel :D